lib/secret_keys/cli.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require "optparse"
+require "io/console"
+
+require_relative "../secret_keys.rb"
+
+class SecretKeys
+  class CLI
+    class Encrypt < CLI
+      def parse_additional_options(opts)
+        @new_secret_key = nil
+        opts.on("--new-secret-key ENCRYPTION_KEY", String, "Encryption key used to encrypt strings in the file on output. This option can be used to change the encryption key.") do |value|
+          @new_secret_key = value
+        end
+      end
+
+      def run
+      end
+    end
+
+    class Decrypt < CLI
+      def run
+      end
+    end
+
+    class Read < CLI
+      def parse_additional_options(opts)
+        @key = nil
+        opts.on("-k", "--key KEY", String, "Key from the file to output. You can use dot notation to read a nested key.") do |value|
+          @key = value
+        end
+      end
+
+      def run
+        raise ArgumentError.new("key is required") if @key.nil? || @key.empty?
+        val = secrets.to_h
+        @key.split(".").each do |key|
+          val = secrets[key] if val.is_a?(Hash)
+        end
+        val
+      end
+    end
+
+    class Edit < CLI
+      def parse_additional_options(opts)
+        @actions = []
+        opts.on("-e", "--set-encrypted KEY=VALUE", String, "Set an encrypted value in the file. You can use dot notation to set a nested value.") do |value|
```

Comment:
```suggestion
        opts.on("-e", "--set-encrypted KEY[=VALUE]", String, "Set an encrypted value in the file. You can use dot notation to set a nested value.") do |value|
```
Might want to mark optional value (if you can use this to mark an existing key as encrypted).